cpu/stm32/kconfigs/f4/Kconfig

```diff
@@ -21,3 +21,18 @@ config HAS_CPU_STM32F4
     bool
     help
         Indicates that the cpu being used belongs to the 'stm32f4' family.
+
+config CLOCK_MAX_84MHZ
+    bool
+    help
+        Indicates that the CPU max core clock frequency is 84MHz
+
+config CLOCK_MAX_100MHZ
+    bool
+    help
+        Indicates that the CPU max core clock frequency is 100MHz
+
+config CLOCK_MAX_180MHZ
+    bool
+    help
+        Indicates that the CPU max core clock frequency is 180MHz
```

Comment:
Why STM32 ? I thought these symbols are only defined when building for STM32 anyway. I can add a prefix for the F4 family, if you prefer.

Reply:
>  I thought these symbols are only defined when building for STM32 anyway.

Yes, they are. I'm just thinking of an scenario where we could try unifying the Kconfig tree (i.e. not depending on `$(BOARD)` and `$(CPU)` on the root Kconfig file). In that case it could make things clearer. Anyways that's not needed for now. I think it's ok to keep it without the family prefix, as others don't have it.